### PR TITLE
DLP Integration Tests

### PR DESCRIPTION
**Pull request by MEseifan**
Some basic integration tests for the DLP, will add more later.

NOTE: I'm not sure which service account is used to run these tests but we need to enable the DLP API for that account so these tests can run

### CODE REVIEW

File: integration-test-remote/src/test/java/io/cdap/cdap/app/etl/gcp/GoogleCloudDataLossPreventionTest.java [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+import com.google.privacy.dlp.v2.Value;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class GoogleCloudDataLossPreventionTest extends DataprocETLTestBase {
+  private static DlpServiceClient client;
```
Comment (shifuxu0301) [inline integration-test-remote/src/test/java/io/cdap/cdap/app/etl/gcp/GoogleCloudDataLossPreventionTest.java:33]: It looks like you are only using the client variable locally, so we can remove static.

File: integration-test-remote/src/test/java/io/cdap/cdap/app/etl/gcp/GoogleCloudDataLossPreventionTest.java [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+  }
+
+  @Override
+  protected void innerTearDown() {
+
+  }
+
+  @BeforeClass
+  public static void setupGoogleDlp() throws Exception {
```
Comment (shifuxu0301) [inline integration-test-remote/src/test/java/io/cdap/cdap/app/etl/gcp/GoogleCloudDataLossPreventionTest.java:58]: I do not think we will need to make this setUp method static?

File: integration-test-remote/src/test/java/io/cdap/cdap/app/etl/gcp/GoogleCloudDataLossPreventionTest.java [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+    }
+    CredentialsProvider provider = new DlpCredentialsProvider(credentials);
+    DlpServiceSettings settings = DlpServiceSettings.newBuilder()
+        .setCredentialsProvider(provider).build();
+    client = DlpServiceClient.create(settings);
+  }
+
+  @Test
+  public void replaceValueTransform() throws Exception {
+
```
Comment (shifuxu0301) [inline integration-test-remote/src/test/java/io/cdap/cdap/app/etl/gcp/GoogleCloudDataLossPreventionTest.java:72]: Nit, remove this extra line.

File: integration-test-remote/src/test/java/io/cdap/cdap/app/etl/gcp/GoogleCloudDataLossPreventionTest.java [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+    String EXPECTED = "I can't believe this news! My SSN is [SSN REDACTED]";
```
Comment (shifuxu0301) [inline integration-test-remote/src/test/java/io/cdap/cdap/app/etl/gcp/GoogleCloudDataLossPreventionTest.java:73]: we can add final since we are not expecting the value change. Also, for best practice, since this variable is not class level, it's better to use lowercase. 

https://www.oracle.com/technetwork/java/javase/documentation/codeconventions-135099.html#367

File: integration-test-remote/src/test/java/io/cdap/cdap/app/etl/gcp/GoogleCloudDataLossPreventionTest.java [hunk 5]
```diff
@@ -0,0 +1,139 @@
...
+    DeidentifyContentResponse deidentifyContentResponse = client.deidentifyContent(request);
+
+    String res = deidentifyContentResponse.getItem().getTable().getRows(0)
+      .getValues(0).getStringValue();
+    Assert.assertEquals(EXPECTED, res);
+  }
+
+  private DeidentifyContentRequest getDeidentifyContentRequestFromPrimitiveTransform(PrimitiveTransformation trans){
+    String REDACTION_TYPE = "US_SOCIAL_SECURITY_NUMBER";
+    String BODY = "I can't believe this news! My SSN is 123-45-6789";
```
Comment (shifuxu0301) [inline integration-test-remote/src/test/java/io/cdap/cdap/app/etl/gcp/GoogleCloudDataLossPreventionTest.java:111]: same comments above about name convention.

File: integration-test-remote/pom.xml [hunk 6]
```diff
@@ -90,6 +90,11 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>com.google.cloud</groupId>
```
Comment (CuriousVini) [inline integration-test-remote/pom.xml:94]: Why is this change needed?
